terraform/sqs.tf.json.template.py

```diff
@@ -12,15 +12,19 @@
         "resource": [
             {
                 "aws_sqs_queue": {
-                    config.unqual_notifications_queue_name(): {
-                        "name": config.notifications_queue_name(),
-                        "visibility_timeout_seconds": config.contribution_lambda_timeout + 10,
-                        "message_retention_seconds": 24 * 60 * 60,
-                        "redrive_policy": json.dumps({
-                            "maxReceiveCount": 10,
-                            "deadLetterTargetArn": "${aws_sqs_queue.%s.arn}"
-                                                   % config.unqual_notifications_queue_name(fail=True)
-                        })
+                    **{
+                        config.unqual_notifications_queue_name(retry=retry): {
+                            "name": config.notifications_queue_name(retry=retry),
+                            "visibility_timeout_seconds": config.contribution_lambda_timeout(retry=retry) + 10,
+                            "message_retention_seconds": 24 * 60 * 60,
+                            "redrive_policy": json.dumps({
+                                "maxReceiveCount": 10,  # TODO increase for retry?
```

Comment:
Same as for aggregates: 1 for the first queue, 9 for the second, totaling 10 attempts as before.